.github/workflows/tests.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: tests
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 0 * * 0' # weekly
```

Comment:
we are running tests on a schedule and not on PR?

Reply:
Ah, so this cron line is to run tests weekly on `main` in addition to PRs. For example, you can see runs on this PR [here](https://github.com/openstax/pro-editing-tool/actions), and examples of the scheduling runs [here](https://github.com/openstax/cnx-epub/actions) in `cnx-epub` (which is what I used as a reference when incorporating that).

Reply:
Oh, i guess the gh actions config file schema is just confusing then...